dango/types/src/config.rs

```diff
@@ -27,3 +41,28 @@ pub struct AppAddresses {
     pub lending: Addr,
     pub oracle: Addr,
 }
+
+/// Default implementation that can be used in tests when the addresses are not
+/// needed.
```

Comment:
This should be a double slash (`//`) instead of triple slash (`///`) because it's not a doc string

Reply:
We can also add `#[cfg(test)]` to this if it's only intended for tests